doc/connectivity/bluetooth/bluetooth-shell.rst

```diff
@@ -0,0 +1,357 @@
+.. _bluetooth_shell:
+
+Bluetooth Shell
+###############
+
+Bluetooth Shell Setup and Usage
+*******************************
+
+First you need to build and flash your board with the Bluetooth shell. For more details on that,
+look at the :ref:`getting_started`. The Bluetooth shell itself is located in
+:zephyr_file:`tests/bluetooth/shell/`.
+
+When it's done, connect to the CLI using your favorite application. You should be welcomed by the
+following prompt:
+
+.. code-block:: console
+
+        uart:~$
+
+For more details on the common usage of the shell, see :ref:`shell_api`.
+
+The first thing you need to do is enable Bluetooth. To do, use :code:`bt init`. Messages should
+confirm you the initialization with a bunch of logging message, if enabled.
```

Comment:
`Messages should be printed confirming Bluetooth has been initialized.`

Could you also add a small snippet of what a `bt init` call ouptuts?